sourmash/_minhash.py

```diff
@@ -585,4 +601,4 @@ def moltype(self):                    # TODO: test in minhash tests
         elif self.hp:
             return 'hp'
         else:
-            return 'DNA'
+            return 'DNA'
```

Comment:
note this line isn't actually changed, it's just had the newline at the end of it removed. something to avoid if possible, since it clutters up the diff :)